### ISSUE TITLE
V-93035 Control failure on Domain Controller

### ISSUE THREAD

**Issue by superyarick** [mitre/microsoft-windows-server-2019-stig-baseline#9]
```   {
              “status”: “failed”,
              “code_desc”: “Control Source Code Error microsoft-windows-server-2019-stig-baseline/controls/V-93035.rb:3 “,
              “run_time”: 0.016248295,
              “start_time”: “2020-07-31T19:27:51+00:00”,
              “message”: “undefined local variable or method `distinguishedName’ for #<Inspec::Rule:0x000055f405b43210>“,
              “exception”: “RuntimeError”,
              “backtrace”: [
                “/var/lib/gems/2.5.0/gems/inspec-core-4.19.2/lib/inspec/rule.rb:61:in `block (2 levels) in initialize’“,
                “/var/lib/gems/2.5.0/gems/rspec-core-3.9.2/lib/rspec/core/example.rb:257:in `instance_exec’“,
                “/var/lib/gems/2.5.0/gems/rspec-core-3.9.2/lib/rspec/core/example.rb:257:in `block in run’“,
                “/var/lib/gems/2.5.0/gems/rspec-core-3.9.2/lib/rspec/core/example.rb:503:in `block in with_around_and_singleton_context_hooks’“,
                “/var/lib/gems/2.5.0/gems/rspec-core-3.9.2/lib/rspec/core/example.rb:460:in `block in with_around_example_hooks’“,
                “/var/lib/gems/2.5.0/gems/rspec-core-3.9.2/lib/rspec/core/hooks.rb:481:in `block in run’“,
                “/var/lib/gems/2.5.0/gems/rspec-core-3.9.2/lib/rspec/core/hooks.rb:619:in `run_around_example_hooks_for’“,
                “/var/lib/gems/2.5.0/gems/rspec-core-3.9.2/lib/rspec/core/hooks.rb:481:in `run’“,
                “/var/lib/gems/2.5.0/gems/rspec-core-3.9.2/lib/rspec/core/example.rb:460:in `with_around_example_hooks’“,
                “/var/lib/gems/2.5.0/gems/rspec-core-3.9.2/lib/rspec/core/example.rb:503:in `with_around_and_singleton_context_hooks’“,
                “/var/lib/gems/2.5.0/gems/rspec-core-3.9.2/lib/rspec/core/example.rb:254:in `run’“,
                “/var/lib/gems/2.5.0/gems/rspec-core-3.9.2/lib/rspec/core/example_group.rb:644:in `block in run_examples’“,
                “/var/lib/gems/2.5.0/gems/rspec-core-3.9.2/lib/rspec/core/example_group.rb:640:in `map’“,
                “/var/lib/gems/2.5.0/gems/rspec-core-3.9.2/lib/rspec/core/example_group.rb:640:in `run_examples’“,
                “/var/lib/gems/2.5.0/gems/rspec-core-3.9.2/lib/rspec/core/example_group.rb:606:in `run’“,
                “/var/lib/gems/2.5.0/gems/rspec-core-3.9.2/lib/rspec/core/example_group.rb:607:in `block in run’“,
                “/var/lib/gems/2.5.0/gems/rspec-core-3.9.2/lib/rspec/core/example_group.rb:607:in `map’“,
                “/var/lib/gems/2.5.0/gems/rspec-core-3.9.2/lib/rspec/core/example_group.rb:607:in `run’“,
                “/var/lib/gems/2.5.0/gems/rspec-core-3.9.2/lib/rspec/core/runner.rb:121:in `block (3 levels) in run_specs’“,
                “/var/lib/gems/2.5.0/gems/rspec-core-3.9.2/lib/rspec/core/runner.rb:121:in `map’“,
                “/var/lib/gems/2.5.0/gems/rspec-core-3.9.2/lib/rspec/core/runner.rb:121:in `block (2 levels) in run_specs’“,
                “/var/lib/gems/2.5.0/gems/rspec-core-3.9.2/lib/rspec/core/configuration.rb:2058:in `with_suite_hooks’“,
                “/var/lib/gems/2.5.0/gems/rspec-core-3.9.2/lib/rspec/core/runner.rb:116:in `block in run_specs’“,
                “/var/lib/gems/2.5.0/gems/rspec-core-3.9.2/lib/rspec/core/reporter.rb:74:in `report’“,
                “/var/lib/gems/2.5.0/gems/rspec-core-3.9.2/lib/rspec/core/runner.rb:115:in `run_specs’“,
                “/var/lib/gems/2.5.0/gems/inspec-core-4.19.2/lib/inspec/runner_rspec.rb:82:in `run’“,
                “/var/lib/gems/2.5.0/gems/inspec-core-4.19.2/lib/inspec/runner.rb:159:in `run_tests’“,
                “/var/lib/gems/2.5.0/gems/inspec-core-4.19.2/lib/inspec/runner.rb:130:in `run’“,
                “/var/lib/gems/2.5.0/gems/inspec-core-4.19.2/lib/inspec/cli.rb:279:in `exec’“,
                “/var/lib/gems/2.5.0/gems/thor-1.0.1/lib/thor/command.rb:27:in `run’“,
                “/var/lib/gems/2.5.0/gems/thor-1.0.1/lib/thor/invocation.rb:127:in `invoke_command’“,
                “/var/lib/gems/2.5.0/gems/thor-1.0.1/lib/thor.rb:392:in `dispatch’“,
                “/var/lib/gems/2.5.0/gems/thor-1.0.1/lib/thor/base.rb:485:in `start’“,
                “/var/lib/gems/2.5.0/gems/inspec-core-4.19.2/lib/inspec/base_cli.rb:35:in `start’“,
                “/var/lib/gems/2.5.0/gems/inspec-bin-4.19.2/bin/inspec:11:in `<top (required)>‘“,
                “/usr/local/bin/inspec:23:in `load’“,
                “/usr/local/bin/inspec:23:in `<main>’”
              ]
            }
          ]
```

**Comment by superyarick** [comment]
Windows Server 2016 does not produce this error. Reference control V-73375

https://github.com/mitre/microsoft-windows-server-2016-stig-baseline/blob/master/controls/V-73375.rb

**Comment by karikarshivani** [comment]
Fixed by #10
